Pressing play after pausing it in now playing restarts video

tombriden opened the pull request:
If you pause the video in now playing and press play again to resume it restarts the video 
Also when paused there's no current time info displayed

Montellese commented:
I can confirm this issue. Took me three attempts yesterday until I realized what was happening.

tombriden committed:
Call updateProgress of now playing screen even when paused so the seek bar times still display
New boolean parameter in updateProgress to determine whether to display play or paused icon

tombriden commented:
The restart when paused issue was already resolved, i can't make it happen with the latest version anyway. The times now display in now playing while paused though.
